app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+
+$f-pnav__main--zindex: 1030;
```

Comment:
z-index 1030 is already a named bootstrap variable: $zindex-navbar-fixed
Is there a reason not to use that?

Reply:
updated